README.md

```diff
@@ -18,3 +18,35 @@ This app allows you to view Kiels public transport (busses, bus-stops) in realti
 ## Screenshots
 
 ![Screenshot](screenshot.jpg)
+
+# Development
```

Comment:
H2 please :stuck_out_tongue_winking_eye:

Reply:
change it